### ISSUE TITLE
md-menu : Throwing error in latest master

### ISSUE THREAD

**Issue by chinnichaitanya** [angular/material#3192]
This is screenshot of the error in the console on clicking any menu button in the latest master.
It seems to be fine in 0.10.0-rc1 though.

![2015-06-10-003302_1366x768_scrot](https://cloud.githubusercontent.com/assets/6941024/8066975/4a166ed8-0f08-11e5-8f6e-3d7bd7c8e74d.png)

**Comment by Frank3K** [comment]
I think it throws [here](https://github.com/angular/material/blob/master/src/components/menu/menu-interim-element.js#L250):

```js
      var containerNode = el[0],
          openMenuNode = el[0].firstElementChild,
          openMenuNodeRect = openMenuNode.getBoundingClientRect(),
          boundryNode = opts.parent[0],
          boundryNodeRect = boundryNode.getBoundingClientRect();
```

**Comment by dohomi** [comment]
Same here, the menu component is broken after 0.10.0-rc2

**Comment by barhaleacct** [comment]
function positionMenu(scope,el, opts) {

Just add scope back in as above, it is missing in the RC2 release, but in the RC1 release

**Comment by fjoubert** [comment]
As @barhaleacct said, I can confirm adding scope back makes the md-menu directive work properly again.

**Comment by Frank3K** [comment]
Should be fixed by fb245fb.

**Comment by chinnichaitanya** [comment]
Yes it is fixed.
